style.css

```diff
@@ -120,3 +120,439 @@
+  font-family: "Inter", serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 16px;
+  line-height: 24px;
+  color: #091e42;
+  top: 280px;
+  position: relative;
+}
+
+.keywords {
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  padding: 0;
+  gap: 5px;
+  height: 40px;
+  width: 316px;
+  position: absolute;
+  top: 440px;
+}
+
+.keywords li {
+  list-style-type: none;
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  padding: 10px 12px;
+  font-family: "Inter", serif;
+  height: 30px;
+  color: #091e42;
+  background: #fff;
+  border: 1px solid #8993a4;
+  font-weight: 400;
+  font-size: 18px;
+}
+
+.button-card1 {
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  padding: 12px;
+  gap: 10px;
+  width: 124px;
+  height: 48px;
+  background: #ff6b00;
+  flex: none;
+  position: absolute;
+  top: 535px;
+  left: 0;
+  font-family: "Inter", serif;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 16px;
+  line-height: 24px;
+  text-align: center;
+  letter-spacing: 0.03em;
+  color: #fff;
+  border: none;
+}
+
+.card {
+  position: absolute;
+  background-image:
+    linear-gradient(
+      180.45deg,
+      rgba(38, 38, 38, 0) -1.25%,
+      rgba(38, 38, 38, 0.9) 123%
+    ),
+    url(images/card-project.png);
+  left: 0;
+  width: 327px;
+  height: 386px;
+}
+
+.card h3 {
+  position: absolute;
+  font-family: "Crete Round", serif;
+  font-size: 30px;
+  color: white;
+  top: 43px;
+  left: 20px;
+  width: 275px;
+}
+
+.card p {
+  position: absolute;
+  width: 295px;
+  height: 87px;
+  font-family: "Inter", serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 16px;
+  line-height: 24px;
+  top: 170px;
+  left: 20px;
+  color: white;
+}
+
+.card ul {
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  padding: 0;
+  gap: 5px;
+  height: 40px;
+  width: 316px;
+  position: absolute;
+  top: 280px;
+  left: 20px;
+}
+
+.card li {
+  list-style-type: none;
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  padding: 10px 12px;
+  font-family: "Inter", serif;
+  height: 30px;
+  color: #fff;
+  background: #ffffff24;
+
+  /* border: 1px solid #8993a4; */
+  font-weight: 400;
+  font-size: 18px;
+}
+
+.card button {
+  position: absolute;
+  height: 48px;
+  left: 0;
+  right: 0;
+  bottom: 0;
+  top: 382px;
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  padding: 12px;
+  padding-top: 20px;
+  padding-bottom: 20px;
+  gap: 10px;
+  background: #ff6b00;
+  border: 0;
+  justify-content: center;
+  font-size: 20px;
+  color: white;
+  font-weight: 600;
+}
+
+.one {
+  top: 783px;
+}
+
+.two {
+  top: 1245px;
+}
+
+.three {
+  top: 1707px;
+}
+
+.four {
+  top: 2169px;
+}
+
+.five {
+  top: 2631px;
+}
+
+.six {
+  top: 3093px;
+}
+
+.about {
+  display: flex;
+  position: absolute;
+  width: 100%;
+  height: 1901px;
+  left: 0;
+  top: 4500px;
+
+  /* Theme 3 / Neutral 1 */
+  background: #1c1a19;
+}
```

Comment:
- [ ] Please 🙏🏽 add space above the about section to me similar to the Figma template 👇🏽 

| Figm template | Your template |
|---|---|
| ![image](https://user-images.githubusercontent.com/58642949/219664325-58aaa6b5-a4ef-4b88-a956-7e0b11b2fbe4.png) | ![image](https://user-images.githubusercontent.com/58642949/219664442-f9fdd388-bae6-483e-b507-40c46d16840b.png) |